src/services/auth/index.tsx

```diff
@@ -172,5 +191,95 @@ export const mockValidateOTP = async (
   };
 };
 
+/** Error message returned by endpoints without a /auth/logout handler */
+const oldBackendError =
+  "Authorization header requires 'Credential' parameter. Authorization header requires 'Signature' parameter. Authorization header requires 'SignedHeaders' parameter. Authorization header requires existence of either a 'X-Amz-Date' or a 'Date' header. Authorization=";
+
+export const liveCallLogout = async (
+  sessionToken: string,
+  operatorToken: string,
+  endpoint: string
+): Promise<void> => {
+  const payload = { sessionToken, operatorToken };
+
+  try {
+    await fetchWithValidator(LogoutResponse, `${endpoint}/auth/logout`, {
+      method: "POST",
+      headers: {
+        Authorization: sessionToken,
+      },
+      body: JSON.stringify({
+        operatorToken,
+      }),
+    });
+    Sentry.addBreadcrumb({
+      category: "logout",
+      message: JSON.stringify({
+        status: "success",
+        ...payload,
+        endpoint,
+      }),
+    });
+    return;
+  } catch (e) {
+    if (e instanceof NetworkError) {
+      throw e;
+    } else if (
+      e instanceof SessionError ||
+      e.message === "Auth token is not currently valid" ||
+      e.message === oldBackendError + sessionToken
+    ) {
+      // for this case the session token is already invalidated or past the valid date
+      // logoutTodo: is it safe to keep session token there if already past validity date
+
+      Sentry.addBreadcrumb({
+        category: "logout",
+        message: JSON.stringify({
+          status: "success despite error",
+          message: e.message,
+          ...payload,
+          endpoint,
+        }),
+      });
+      return;
+    } else if (
+      e.message === "Auth tokens are mismatched" ||
+      e.message === "Auth token is missing"
+    ) {
+      // these shouldn't happen because the operator token should be correct
+      Sentry.addBreadcrumb({
+        category: "logout",
+        message: JSON.stringify({
+          status: "operator token error",
+          message: e.message,
+          ...payload,
+          endpoint,
+        }),
+      });
+      throw new OperatorTokenError(e.message);
+    } else {
+      Sentry.addBreadcrumb({
+        category: "logout",
+        message: JSON.stringify({
+          status: "failed",
+          message: e.message,
+          ...payload,
+          endpoint,
+        }),
+      });
+      throw new LogoutError(e.message);
+    }
+  }
+};
+
+export const mockCallLogout = async (
+  _sessionToken: string,
+  _operatorToken: string,
+  _endpoint: string
+): Promise<void> => {
+  // throw new LogoutError("error logging out");
```

Comment:
Is this intended?

Reply:
yep it's deliberately commented out, it's for if a failed Logout needs to be tested during mocking.

Reply:
Can we leave a comment above to explain? Something like 

```
// Uncomment to test throwing an error
// throw new LogoutError("error logging out");
```